crates/api_common/README.md

```diff
@@ -24,3 +24,10 @@ As you can see, each API endpoint needs a parameter type ( GetPosts), path (/pos
 For a real example of a Lemmy API client, look at [lemmyBB](https://github.com/LemmyNet/lemmyBB/tree/main/src/api).
 
 Lemmy also provides a websocket API. You can find the full websocket code in [this file](https://github.com/LemmyNet/lemmy/blob/main/src/api_routes_websocket.rs).
+
+## Generate TypeScript bindings
+
+TypeScript bindings (API types) can be generated by running `cargo test --features full`.
```

Comment:
Mention that this needs to be done inside `crates/api_common` folder. Alternatively you can run `cargo test -p lemmy_api_common --features full` from root.

Reply:
In that case make the command this to be more clear.

```
cd crates/api_common
cargo test ...
```